packages/camera/camera/README.md

```diff
@@ -16,6 +16,12 @@ A Flutter plugin for iOS, Android and Web allowing access to the device cameras.
 * Snapshots can be captured and saved to a file.
 * Record video.
 * Add access to the image stream from Dart.
+* Zoom camera
+* Flash support
+* Auto Focus
```

Comment:
Please be consistent about capitalization; this is using title case unlike all the other entries.